### ISSUE TITLE
Add FCLCrypto to default deployed contracts

### ISSUE THREAD

**Issue by gregsantos** [onflow/flow-emulator#159]
### FCLCrypto contract should be deployed along with default contracts on emulator start

Dev wallet relies on `FCLCrypto Contract` and currently needs to be manually deployed to avoid an error
when authenticating with FCL ^1.0.0

If added as one of the default contracts on emulator start, this step and error can be avoided.

https://github.com/onflow/fcl-contracts/tree/main/contracts

**Comment by sideninja** [comment]
@gregsantos sorry for late reply, should we prioritize this in this sprint?

**Comment by gregsantos** [comment]
No problem @sideninja 
Yea, i think it is a good idea to prioritize this sprint as it will address a prominent issue with local development.

**Comment by bluesign** [comment]
why this contract exists btw ? it seems like it is just one function.

**Comment by sideninja** [comment]
@gregsantos I'm finishing up this task, but I would need the https://github.com/onflow/fcl-contracts to become public repo.

**Comment by gregsantos** [comment]
@psiemens may need to open as code owner

@bluesign This contract is used in verifying signatures (user, and account-proof)

**Comment by bluesign** [comment]
I understand @gregsantos but as I mentioned earlier[0], especially emulator has so many unnecessary contracts. 

To run a single function, deploying a new contract, seems little unnecessary to me. Is there a specific reason for this to be a contract, instead of Function in the script ?

[0] https://github.com/onflow/flow-emulator/issues/170

**Comment by gregsantos** [comment]
@bluesign Thanks Deniz, your thoughts on emulator contracts makes sense. 
We created the FCLCrypto contract when the cadence Crypto lib was removed and to help with transition to account-proof v2. We may switch back to using a script, making this unnecessary soon. For now it might be better to deploy the contract along with others in the dev-wallet. Closing this in favor of https://github.com/onflow/fcl-dev-wallet/issues/136
